### PR TITLE
Create Rule5-50.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section5/Rule5-50.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- Case 1: If shading by adjacent structure and terrain is modeled the same in B-RMR as in P-RMR: `if B_RMR.building.has_site_shading == P_RMR.building.has_site_shading: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Notes:**
+
+1. Did not find building.has_site_shading in the current schema.
```
Comment (mkarpman) [inline docs/section5/Rule5-50.md:30]: I have proposed on github as issue #31 for the schema

File: docs/section5/Rule5-50.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** None  
+
+## Rule Logic:  
+
+**Rule Assertion:**  
+
+- Case 1: If shading by adjacent structure and terrain is modeled the same in B-RMR as in P-RMR: `if B_RMR.building.has_site_shading == P_RMR.building.has_site_shading: PASS`
+
```
Comment (mkarpman) [inline docs/section5/Rule5-50.md:23]: We should have three options here: (1) PASS if there is no site shading in either b_RMR or P-RMR, (2) Manual check if there is site shading in both P-RMR and B-RMR, to verify that it's the same, and (3) fail if there is site shading in B-RMR is not equal to P-RMR

File: docs/section5/Rule5-50.md (hunk 1 above)
Comment (xingcx) [reply docs/section5/Rule5-50.md:30]: Updated.

File: docs/section5/Rule5-50.md (hunk 2 above)
Comment (xingcx) [reply docs/section5/Rule5-50.md:23]: Updated.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**